File: 1-TwoSum/1-TwoSum.cpp
// Last updated: 06/07/2026, 21:55:52
class Solution {
public:
    vector<int> twoSum(vector<int>& nums, int target) {
        int size=nums.size();
        for(int i=0;i<size;i++){
            for(int j=i+1;j<size;j++){
                if(nums[i]+nums[j]==target){
                    return {i,j};
                }

            }
        }
        return {0};
    }
};

File: 1-TwoSum/1-TwoSum.py
# Last updated: 06/07/2026, 21:55:50
class Solution(object):
    def twoSum(self, nums, target):
        i=0
        mp={}
        
        for num in range(0,len(nums)):
            rem=target-nums[num]

            if rem in mp:
                return (mp[rem],num)
            mp[nums[num]]=num

            
        